docs/src/pages/ui/theme/alternativeStyling/index.page.mdx

```diff
@@ -0,0 +1,501 @@
+
+##### Amplify Custom ClassNames
+
+These are custom classnames added onto Amplify components and can be used with CSS styling rules to modify the look of an Amplify component.
+
+```jsx
+/** @jsxImportSource @emotion/react */
+import { jsx, css } from '@emotion/react';
+import { Text } from '@aws-amplify/ui-react';
+
+const styles = {
+  color: 'blue',
+};
+
+<Text css={css(styles)} className="my-styled-text">
+  This is my Emotion Text{' '}
+</Text>;
+```
+
+```CSS
+/* External Style Sheet */
+.my-styled-text {
+  color: red;
+}
+```
+
+In the example above, the color of the text will be `blue` because the Emotion styling will take precedence over the classname CSS.
+
+##### CSS Root Variable Overrides
+
+Amplify theming provides many CSS variables to customize the look and feel of the entire application. These CSS variables can be used at a root level to override all instances of component styling.
+
+```jsx
+/** @jsxImportSource @emotion/react */
+import { jsx, css } from '@emotion/react';
+import { Button } from '@aws-amplify/ui-react';
+
+const styles = {
+  color: 'blue',
+};
+
+<Button css={css(styles)}>Button</Button>;
+```
+
+```CSS
+/* External Style Sheet */
+:root {
+  --amplify-components-button-color: red;
+}
+```
+
+In the example above, the button color will be set to `blue` because the Emotion styling will take precedence over the Amplify CSS variable.
+
+</Block>
+
+<Block name="Aphrodite">
+[Aphrodite](https://github.com/Khan/aphrodite) is a Framework-agnostic CSS-in-JS with support for server-side rendering, browser prefixing, and minimum CSS generation.
+
+#### Usage
+
+```jsx
+import { StyleSheet, css } from 'aphrodite';
+import { Text } from '@aws-amplify/ui-react';
+
+const styles = StyleSheet.create({
+  blue: {
+    color: 'blue',
+  },
+});
+
+//use the Aphrodite css function and the className prop
+<Text className={css(styles.blue)}>This is an aphrodite Text component</Text>;
+```
+
+#### Interactions
+
+There are various ways to customize an Amplify component and Aphrodite will interact with these customizations in the following ways:
+
+##### Amplify Styling Props
+
+These are styling props that can be used directly on an Amplify component which will affect a single style property. Examples of styling props are `color` or `fontWeight`. Aphrodite will take precedence over these Amplify styling props.
+
+```jsx
+import { StyleSheet, css } from 'aphrodite';
+import { Text } from '@aws-amplify/ui-react';
+
+const styles = StyleSheet.create({
+  blue: {
+    color: 'blue',
+  },
+});
+
+<Text className={css(styles.blue)} color="red">
+  This is an aphrodite Text component
+</Text>;
+```
+
+In the example above, the color of the view will be set to `blue` because of the Aphrodite styling will take precedence. [Here]() is a list of all Amplify styling props.
```

Comment:
It looks like this is the only library where this use-case will be 'blue' instead of 'red', is that correct?

Reply:
yes by default aphrodite adds !important to all of their styling, there is a config option to turn this off but I just have the default behavior covered here.

Reply:
ohhh interesting.... should we make a note of this in the docs?

Reply:
yes we should probably make a note of this, what I didn't want to do is write two docs for Aphrodite where one is how this behaves by default and the other is how this behaves if you take off !important.  Should I just make a note that this is happening because of !important or should I also note that Aphrodite can be configured to take !important off in which case it behaves more like the others?

Reply:
Yea I think just make a note about the behavior and how they can change that with the aphrodite config.